app/bundles/LeadBundle/Tests/EventListener/CampaignSubscriberFunctionalTest.php

```diff
@@ -884,6 +927,157 @@ private function createCampaignWithPointEvents(array $contactIds, int $pointGrou
+            $campaignLead->setCampaign($campaign);
+            /** @var Lead $lead */
+            $lead = $this->em->getReference(Lead::class, $contactId);
+            $campaignLead->setLead($lead);
+            $campaignLead->setDateAdded(new \DateTime());
+            $this->em->persist($campaignLead);
+            $campaign->addLead($key, $campaignLead);
+        }
+
+        $this->em->flush();
+
+        $event = new Event();
+        $event->setCampaign($campaign);
+        $event->setName('Update contact with tokens');
+        $event->setType('lead.updatelead');
+        $event->setEventType('action');
+        $event->setTriggerMode('immediate');
+        $event->setProperties(
+            [
+                'canvasSettings'             => [
+                    'droppedX' => '696',
+                    'droppedY' => '155',
+                ],
+                'name'                       => '',
+                'triggerMode'                => 'immediate',
+                'triggerDate'                => null,
+                'triggerInterval'            => '1',
+                'triggerIntervalUnit'        => 'd',
+                'triggerHour'                => '',
+                'triggerRestrictedStartHour' => '',
+                'triggerRestrictedStopHour'  => '',
+                'anchor'                     => 'leadsource',
+                'properties'                 => [
+                    'html'                 => '',
+                    'title'                => '',
+                    'html2'                => '',
+                    'firstname'            => '',
+                    'lastname'             => '',
+                    'company'              => '',
+                    'position'             => '',
+                    'email'                => '',
+                    'mobile'               => '',
+                    'phone'                => '',
+                    'points'               => '',
+                    'fax'                  => '',
+                    'address1'             => '',
+                    'address2'             => '',
+                    'city'                 => '',
+                    'state'                => '',
+                    'zipcode'              => '',
+                    'country'              => '',
+                    'preferred_locale'     => '',
+                    'timezone'             => '',
+                    'last_active'          => '',
+                    'attribution_date'     => '',
+                    'attribution'          => '',
+                    'website'              => '',
+                    'facebook'             => '',
+                    'foursquare'           => '',
+                    'instagram'            => '',
+                    'linkedin'             => '',
+                    'skype'                => '',
+                    'twitter'              => '',
+                ],
```

Comment:
Let's remove this. I think it's still in the DB due to backward compatibility. Let's remove everything that is not necessary for the test.